server/mysterium_api.go

```diff
@@ -0,0 +1,122 @@
+package server
```

Comment:
I know this is probably the best naming we can come up with, but maybe worth naming it discovery_api? mysterium_api is pretty generic:)

Reply:
Well from node/client perspective this api provides 3 distinct functionalities:
1. Register identities
2. Register/provide proposals (actual discovery)
3. Collect statistics
We call remote service "discovery service" as we don't have a better naming yet. But from client/node perspective - its not discovery but something more generic - i.e. mysterium_centralized_api